.gitignore

```diff
@@ -1,9 +1,12 @@
 /target/
 **/*.rs.bk
 Notion.msi
```

Comment:
Remove the `Notion.*` ignored files here

Reply:
Good catch—but I actually left this in place on purpose so that if someone happens to have them hanging around after a rename locally, they don't accidentally check them in!

Reply:
ah gotcha